source/client/client_worker_impl.h

```diff
@@ -31,7 +31,7 @@ class ClientWorkerImpl : public WorkerImpl, virtual public ClientWorker {
                    const RequestSourceFactory& request_generator_factory,
                    Envoy::Stats::Store& store, const int worker_number,
                    const Envoy::MonotonicTime starting_time,
-                   Envoy::Tracing::HttpTracerPtr& http_tracer);
+                   Envoy::Tracing::HttpTracerPtr& http_tracer, const bool simple_warmup);
```

Comment:
How will this API evolve? This is about my usual concern around readability of boolean flags. Is this related to phases and will this be removed in favor of some phase information? Or should we introduce an enum now?

Reply:
See 5940d5601eaa9924782c57be5a67895a7a1dfdd5, let me know what you think.